diff --git a/dist/index.js b/dist/index.js
index 7ed1bef23d7263de192c3b60bcfe8861bb2f0053..35903b46eae0a268f54d82f42573f1f0e8c2a550 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-"use strict"; const crypto = require("crypto")["webcrypto"]; const secp256k1Zkp = (typeof WebAssembly !== "undefined") ? require("./wasm.js") : require("./asm.js");
+"use strict"; const crypto = require("crypto")["webcrypto"]; const secp256k1Zkp = (typeof WebAssembly !== "undefined" && false) ? require("./wasm.js") : require("./asm.js");
 // Use strict
 "use strict";
 