diff --git a/dist/index.js b/dist/index.js
index c6efe6cfaa9db2a1a199fac142560c30239c5675..c77b13a1ae9dc51031659486c4553b8f590a1758 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-"use strict"; const x25519 = (typeof WebAssembly !== "undefined") ? require("./wasm.js") : require("./asm.js");
+"use strict"; const x25519 = (typeof WebAssembly !== "undefined" && false) ? require("./wasm.js") : require("./asm.js");
 // Use strict
 "use strict";
 